src/configuration.ts

```diff
@@ -38,8 +39,8 @@ const configuration = Object.freeze({
         chains: [ChainId.ETHEREUM_MAINNET],
         optionalChains: [ChainId.MATIC_MAINNET]
       },
-      [ChainId.ETHEREUM_GOERLI]: {
-        chains: [ChainId.ETHEREUM_GOERLI],
+      [ChainId.ETHEREUM_SEPOLIA]: {
+        chains: [ChainId.ETHEREUM_SEPOLIA],
```

Comment:
Instead of replacing Goerli, Add Sepolia as another chain.

Create an issue for the future to remove Goerli from the supported chains.

Reply:
Add sepolia as an optional chain instead.